libs/langchain/langchain/retrievers/google_cloud_enterprise_search.py

```diff
@@ -9,14 +9,14 @@
 from langchain.schema import BaseRetriever, Document
 from langchain.utils import get_from_dict_or_env
 
+from google.api_core.exceptions import InvalidArgument
```

Comment:
This is an optional package and may not be installed for users -- the import needs to be tucked into the appropriate place

Reply:
resolved